pkg/application/factory/manager/manager.go

```diff
@@ -1,15 +1,17 @@
 package factory
 
+import (
+	contextPkg "github.com/codeready-toolchain/sandbox-auth/pkg/application/service/context"
```

Comment:
Let's not use camelCase for package names as the golang convention recommends: https://go.dev/blog/package-names